### ISSUE TITLE
gpu

### ISSUE THREAD

**Issue by roar-1128** [choyingw/GAIS-Net#5]
请问作者单卡可以跑吗

**Comment by choyingw** [comment]
Yes, you can run on a single GPU with a smaller batch size that fits in
your GPU card (and modify learning rate correspondingly). Please check the
config file for those two hyperparameter.



Ph.D. candidate of Computer Science Department
University of Southern California
